frontend/src/locales/en.json

```diff
@@ -1375,7 +1375,7 @@
   "app": {
     "cantView": "You can't currently view the app right now. You can try clearing LocalStorage to reset your settings.",
     "warning": {
-      "title": "Metamask Not Detected Or Incorrect Network",
+      "title": "Metamask Not Detected",
```

Comment:
Warning only appears now, if the user has no MM or another provider installed

Reply:
Can this be changed to "No Wallet Detected" Since metamask is no longer relevant

Reply:
Would leave it at that for now. You can connect any wallet, but our onboarding links to MM install. Also, I haven't tested all ins and outs of the bsc wallet or coinbase wallet and if they accept our network change and config